src/CommunityToolkit.Maui.Core/Primitives/MathOperator.shared.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace CommunityToolkit.Maui.Core;
+
+/// <summary>
+/// Precedence for Math Opertator
+/// </summary>
+public enum MathOperatorPrecedence
```

Comment:
Yup - that's cool. I'll create an `Enums` folder.

Reply:
Hey Vlad! I started the process of creating an `Enums` folder and moving all of the enums to it. 

But then I quickly realized that this would require a new `xmlns`, making life harder for XAML devs.

I agree that we should move the files, but I think we should wait until #144 is completed to avoid making life hard for XAML devs before #144 makes it simpler.

I've added this task to a Proposal #219, which we'll complete after #144 👍